packages/frontend/stalker-app/src/app/modules/websites/view-website/website-overview/website-overview.component.ts

```diff
@@ -0,0 +1,122 @@
+import { MatProgressSpinnerModule } from '@angular/material/progress-spinner';
+import { MatSidenavModule } from '@angular/material/sidenav';
+import { MatTableModule } from '@angular/material/table';
+import { MatTooltipModule } from '@angular/material/tooltip';
+import { BehaviorSubject, combineLatest, debounceTime, filter, map, shareReplay, Subject, switchMap, tap } from 'rxjs';
+import { FindingsService } from '../../../../api/findings/findings.service';
+import { SharedModule } from '../../../../shared/shared.module';
+import { Website } from '../../../../shared/types/websites/website.type';
+import { TextMenuComponent } from '../../../../shared/widget/text-menu/text-menu.component';
+import { FindingsModule } from '../../../findings/findings.module';
+
+@Component({
+  standalone: true,
+  imports: [
+    CommonModule,
+    SharedModule,
+    MatCardModule,
+    MatIconModule,
+    MatListModule,
+    MatFormFieldModule,
+    MatTableModule,
+    MatPaginatorModule,
+    MatSidenavModule,
+    MatButtonModule,
+    MatInputModule,
+    MatProgressSpinnerModule,
+    MatMenuModule,
+    FormsModule,
+    FindingsModule,
+    MatTooltipModule,
+    TextMenuComponent,
+    FindingsModule,
+  ],
+  selector: 'app-website-overview',
+  templateUrl: './website-overview.component.html',
+  styleUrls: ['./website-overview.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class WebsiteOverviewComponent {
+  public websiteSubject$: BehaviorSubject<Website | undefined> = new BehaviorSubject<Website | undefined>(undefined);
+  public website$ = this.websiteSubject$.pipe(filter((w) => !!w));
+  public _website!: Website;
+
+  @Input()
+  public set website(wsite: Website) {
+    console.log('website setter');
+    this.websiteSubject$.next(wsite);
+    this._website = wsite;
+  }
+
+  public get website() {
+    return this._website;
+  }
+
+  public a = this.website$.pipe(
+    tap((w) => {
+      console.log('website tap');
+      console.log(w);
+    })
+  );
+
+  public sitemapFilterChange$ = new BehaviorSubject<string>('');
+  public selectedEndpoint: string = '';
+  public endpointLoading: boolean = false;
+  public selectedEndpoint$ = new Subject<string>();
+  public endpointData$ = combineLatest([this.selectedEndpoint$, this.website$]).pipe(
+    tap(() => {
+      this.endpointLoading = true;
+    }),
+    debounceTime(200),
+    switchMap(([endpoint, website]) => {
+      this.selectedEndpoint = endpoint;
+      return this.findingService.getLatestWebsiteEndpoint(website!.correlationKey, endpoint);
+    }),
+    tap(() => {
+      this.endpointLoading = false;
+    }),
+    shareReplay(1)
+  );
+
+  public sitemap$ = combineLatest([this.website$, this.sitemapFilterChange$]).pipe(
+    map(([website, filter]) => {
+      console.log('coucou!!');
+      return website!.sitemap.filter((v) => v.toLocaleLowerCase().includes(filter.toLocaleLowerCase()));
+    }),
+    tap(() => this.cdr.detectChanges())
```

Comment:
Est-ce que ceci est nécessaire? On ne semble pas muter de données dans cette pipe (which is good), alors le `| async` devrait suffisant pour que la vue s'update. Si cette ligne est nécessaire, j'ai l'impression que c'est un hasard!

Reply:
indeed c'etait pas necessaire, bien vu!